code/game/objects/items/tanks/jetpack.dm

```diff
@@ -248,7 +249,7 @@
 
 /mob/living/carbon/human/get_jetpack()
 	var/obj/item/tank/jetpack/J = ..()
-	if(!istype(J) && istype(wear_suit, /obj/item/clothing/suit/space/hardsuit))
+	if(!istype(J) && (istype(wear_suit, /obj/item/clothing/suit/space/hardsuit) || istype(wear_suit, /obj/item/clothing/suit/hooded/wintercoat/cosmic/true)))
 		var/obj/item/clothing/suit/space/hardsuit/C = wear_suit
 		J = C.jetpack
```

Comment:
Hacky code. You are checking if an atom is of a given type just to assign it to a var of a different non-parent type.

Reply:
I mean it works since they're calling the same vars

Reply:
But I see what ya mean